WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -53,7 +70,13 @@ extension SiteAddressViewController: UITableViewDataSource {
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        return tableView.dequeueReusableCell(withIdentifier: "TextFieldCell") ?? UITableViewCell()
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: InstructionTableViewCell.reuseIdentifier, for: indexPath) as? InstructionTableViewCell else {
+            fatalError()
```

Comment:
Can this just `return  UITableViewCell()`?

Reply:
sure :)